### ISSUE TITLE
Give buttons outlines so they stand out in high-contrast mode

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1463]
Some browsers and OSes let users turn on high-contrast mode, which appears to make the screen entirely black and white:

![CivilForm1](https://user-images.githubusercontent.com/77128518/123179732-d5811700-d43e-11eb-9e99-1463050a5a7b.PNG)

Here, you can see that our buttons (Register, etc.) are supposed to have a solid background with no border/outline, and so in high-contrast mode they blend into the background.

To make the buttons more visually distinct (and ensure they still look like buttons), we should make sure our buttons all have borders or outlines.

## Testing method

Turn on High Contrast Mode on Safari (on Mac, I would guess).

## Original tester's report

> ACTUAL RESULT:  'Continue' button looks like normal text in High contrast mode which can be confusing for the user.
> EXPECTED RESULT: Items in the page should be marked properly in High Contrast mode. Continue button should have a border to demarcate it as such and should not be visible as plain text.
